src/main/java/AWSClientFactory.java

```diff
@@ -58,12 +62,15 @@ public AWSClientFactory(String proxyHost, String proxyPort, String awsAccessKey,
         Validation.checkAWSClientFactoryRegionConfig(region);
         this.region = region;
 
-        clientConfig = new ClientConfiguration();
-        clientConfig.setUserAgentPrefix("CodeBuild-Jenkins-Plugin"); //tags all calls made from Jenkins plugin.
         Validation.checkAWSClientFactoryProxyConfig(proxyHost, proxyPort);
-        clientConfig.setProxyHost(proxyHost);
+        clientConfig = new ClientConfiguration()
+                .withUserAgentPrefix("CodeBuild-Jenkins-Plugin") //tags all calls made from Jenkins plugin.
+                .withProxyHost(proxyHost)
+                .withRetryPolicy(new RetryPolicy(new CodeBuildClientRetryCondition(),
+                        new PredefinedBackoffStrategies.ExponentialBackoffStrategy(5000, 20000),
```

Comment:
These values worked for us when we have around 100 concurrent builds running. Anything less still threw up `ThrottlingException`